js/apps/system/_admin/aardvark/APP/react/src/views/databases/viewDatabase/SingleDatabaseView.tsx

```diff
@@ -0,0 +1,66 @@
+import { ArrowBackIcon, DeleteIcon } from "@chakra-ui/icons";
+import { Button, Flex, Heading, IconButton, Stack } from "@chakra-ui/react";
+import React from "react";
+import { useHistory, useRouteMatch } from "react-router-dom";
+import { DatabaseInfo } from "./DatabaseInfo";
+import { DeleteDatabaseModal } from "./DeleteDatabaseModal";
+import { useFetchDatabase } from "../useFetchDatabase";
+
+export const SingleDatabaseView = () => {
+  const { params } = useRouteMatch<{ databaseName: string }>();
+  const { database: currentDatabase } = useFetchDatabase(params.databaseName);
+  const [showDeleteModal, setShowDeleteModal] = React.useState(false);
+  const history = useHistory();
+  if (!currentDatabase) {
+    return null;
+  }
+  const currentDbName = window.frontendConfig.db;
+  const isSystem = currentDatabase.name === "_system";
+  const isSameDb = currentDatabase.name === currentDbName;
+  const showDeleteButton = !isSystem && !isSameDb;
+
+  return (
+    <Stack padding="6" width="100%" height="calc(100vh - 120px)">
+      <Flex direction="row" alignItems="center">
+        <IconButton
+          aria-label="Back"
+          variant="ghost"
+          size="sm"
+          icon={<ArrowBackIcon width="24px" height="24px" />}
+          onClick={() => {
+            history.push("/databases");
+          }}
+        />
+        <Heading marginLeft="2" as="h1" size="lg">
+          Database: {params.databaseName}
```

Comment:
since database name can be large, this can cause the UI to break:


to fix, we can either use a `<Grid>` as the container, or pass `wordBreak=break-all`.

(also need to do `flexShrink: 0` for the 'delete' button)

Also check `isTruncated` prop for `<Heading>`, in case it's useful (if truncating, I'd recommend adding `title` prop too)


<img width="1320" alt="image" src="https://github.com/arangodb/arangodb/assets/2976363/acd4bba9-2509-41ef-bbb7-b982c5cb3a39">

<img width="1313" alt="image" src="https://github.com/arangodb/arangodb/assets/2976363/0f6d034c-f824-4992-8440-e86ff6423c5e">

Reply:
Yes, it probably is. This database was already stored in my local so noticed it.